stanford/lagunita/group_vars/all

```diff
@@ -15,8 +15,7 @@ CERTS_SITE_DISCLAIMER_TEXT: '<b>PLEASE NOTE:</b> SOME ONLINE COURSES MAY DRAW ON
 CLUSTER_NUMBER: 1
 COMMON_DEPLOYMENT: lagunita
 COMMON_DEPLOYMENT_DIR: "{{ secure_dir }}/vars/{{ COMMON_DEPLOYMENT }}"
-COMMON_HOSTNAME: "{{ COMMON_DEPLOYMENT }}-box-{{ CLUSTER_NAME }}-{{ CLUSTER_NUMBER }}"
-# COMMON_HOSTNAME: "{{ CLUSTER_NAME }}{{ CLUSTER_NUMBER }}.{{ COMMON_DEPLOYMENT }}.vpc"
+COMMON_HOSTNAME: "{{ CLUSTER_NAME }}{{ CLUSTER_NUMBER }}.{{ COMMON_DEPLOYMENT }}.vpc"
```

Comment:
Use hostnames that match the private DNS, like:

- `rabbitmq1.stage.vpc`
instead of
- `stage-box-rabbitq-1`